src/MsGraph.php

```diff
@@ -94,13 +94,20 @@ public function connect($id = null)
                     'token_id' => $result->id,
                     'info' => $me
                 ];
-
```

Comment:
this is been added in now and can be removed from your PR

Reply:
File should be the same now